mapviz_plugins/src/odometry_plugin.cpp

```diff
@@ -198,18 +141,17 @@ namespace mapviz_plugins
     stamped_point.stamp = odometry->header.stamp;
     stamped_point.source_frame = odometry->header.frame_id;
 
-    stamped_point.point = tf::Point(
-        odometry->pose.pose.position.x,
-        odometry->pose.pose.position.y,
-        odometry->pose.pose.position.z);
+    stamped_point.point = tf::Point(odometry->pose.pose.position.x,
+                                    odometry->pose.pose.position.y,
+                                    odometry->pose.pose.position.z);
 
     stamped_point.orientation = tf::Quaternion(
-        odometry->pose.pose.orientation.x,
-        odometry->pose.pose.orientation.y,
-        odometry->pose.pose.orientation.z,
-        odometry->pose.pose.orientation.w);
+        odometry->pose.pose.orientation.x, odometry->pose.pose.orientation.y,
+        odometry->pose.pose.orientation.z, odometry->pose.pose.orientation.w);
```

Comment:
This re-formatting makes this call significantly harder to read. Once an argument list wraps onto multiple lines, it is almost always preferable to split every argument onto its own line.